torch/_decomp/__init__.py

```diff
@@ -15,7 +15,7 @@
 meta_lib = torch.library.Library("aten", "IMPL", "Meta")
 
 
-def register_decomposition(aten_op, registry=None, *, register_meta: bool = False):
+def register_decomposition(aten_op, registry=None, *, disable_meta: bool = False):
```

Comment:
(no action required) ack, people are going to add disable_* flags for every subsystem as they come along. Not sure if that's the UX we want